### DISCUSSION TITLE
ebsd data import problem

### DISCUSSION THREAD

**Discussion by 155511sd** [mtex-toolbox/mtex#1650] [Ask Anything]
When importing ebsd data (crc file), the display is not recognized, you can ask for help in the forum

**Comment by lfgmorales** [comment]
Do you have both *.crc and *.cpr? You need both in files coming from Oxford. Alternatively, export a *.ctf, which is a text file

Cheers


Liiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 13 Nov 2022, at 15:16, 155511sd ***@***.***> wrote:

﻿

When importing ebsd data (crc file), the display is not recognized, you can ask for help in the forum

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1650>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN23FOIKJXMUB5JYFZTWIDZ2ZANCNFSM6AAAAAAR7BTBB4>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1650", "url": "https://github.com/mtex-toolbox/mtex/discussions/1650", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by 155511sd** [comment]
This problem is solved，thanks！


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mtex-toolbox/mtex"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月13日(星期天) 晚上10:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [mtex-toolbox/mtex] ebsd data import problem (Discussion #1650)





 Do you have both *.crc and *.cpr? You need both in files coming from Oxford. Alternatively, export a *.ctf, which is a text file 
 
 Cheers 
 
 
 Liiz 
 
 Luiz F. G. Morales 
 ScopeM - ETH Zürich 
 Sent from iPhone 
 
 On 13 Nov 2022, at 15:16, 155511sd ***@***.***&gt; wrote: 
 
 ﻿ 
 
 When importing ebsd data (crc file), the display is not recognized, you can ask for help in the forum 
 
 — 
 Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1650&gt;, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN23FOIKJXMUB5JYFZTWIDZ2ZANCNFSM6AAAAAAR7BTBB4&gt;. 
 You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt; 
 
 [ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1650", "url": "https://github.com/mtex-toolbox/mtex/discussions/1650", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
